### PR TITLE
feat: [FFM-7651]: hide Git Management nav link when GITEX flag enabled

### PR DESCRIPTION
**Pull request by Kira-Jennings**
### Summary
Flag `FF_FLAG_SYNC_THROUGH_GITEX_ENABLED` takes priority over `FF_GITSYNC`

#### Screenshots
PROJECT SETUP

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by laura-mannus — COMMENTED** [review]
I like it :D

**Review by laura-mannus — APPROVED** [review]

File: src/modules/75-cf/components/SideNav/SideNav.tsx [hunk 1]
```diff
@@ -90,21 +90,25 @@ export default function CFSideNav(): React.ReactElement {
...
-              {FF_GITSYNC && !isGitSimplificationEnabled && (
-                <>
-                  <SidebarLink
-                    label={getString('connectorsLabel')}
-                    to={routes.toConnectors({ ...params, module: 'cf' })}
-                  />
-                  <SidebarLink label={getString('common.secrets')} to={routes.toSecrets({ ...params, module: 'cf' })} />
-                  {!FF_FLAG_SYNC_THROUGH_GITEX_ENABLED && (
+              {(FF_GITSYNC && !isGitSimplificationEnabled) ||
+                (FF_FLAG_SYNC_THROUGH_GITEX_ENABLED && isGitSimplificationEnabled && (
```
Comment (knagurski) [inline src/modules/75-cf/components/SideNav/SideNav.tsx:94]: Looks like your parens are off
```suggestion
              {((FF_GITSYNC && !isGitSimplificationEnabled) ||
                (FF_FLAG_SYNC_THROUGH_GITEX_ENABLED && isGitSimplificationEnabled)) && (
```

Comment (Kira-Jennings) [reply src/modules/75-cf/components/SideNav/SideNav.tsx:94]: The linter drops them

Comment (Kira-Jennings) [reply src/modules/75-cf/components/SideNav/SideNav.tsx]: Nope I'm an idiot, fixed

**Review by laura-mannus — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
💖

**Review by Harness-JKhattak — APPROVED** [review]

**Pull request merged**